ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/DomainSpecific/Paged/BraveSearchResultAdScript.js

```diff
@@ -46,6 +56,10 @@ window.__firefox__.includeOnce('BraveSearchResultAdScript', function($) {
         jsonLd.creatives.forEach(creative => {
           if (creative['@type'] === 'SearchResultAd') {
             let jsonLdCreative = {};
+            let placementId = creative['data-placement-id'];
+            if (!placementId || !checkIsAdVisible(placementId)) {
```

Comment:
nit: `isAdPlaced`?

Reply:
the ad can be placed but not visible, so I'd go with isAdVisible